investigator/investigator/package_released/investigate_package_released.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# thoth-investigator
+# Copyright(C) 2020 Bissenbay Dauletbayev
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This file contains methods used by Thoth investigator to investigate on package releases."""
+
+from thoth.storages.graph import GraphDatabase
+from thoth.messaging import MessageBase
+from thoth.messaging import PackageReleasedMessage
+from thoth.common import OpenShift
+
+from ..metrics import scheduled_workflows
+from .. import common
+from .metrics_package_released import package_released_exceptions
+from .metrics_package_released import package_released_in_progress
+from .metrics_package_released import package_released_success
+
+
+@package_released_exceptions.count_exceptions()
+@package_released_in_progress.track_inprogress()
+def parse_package_released_message(package_released: MessageBase, openshift: OpenShift, graph: GraphDatabase) -> None:
+    """Parse package released message."""
+    package_name = package_released.package_name
+    package_version = package_released.package_version
+    index_url = package_released.index_url
+
+    # Solver logic
+
+    common.learn_using_solver(
```

Comment:
you need to take the outputs of these methods, which are the number of workflows scheduled per type to be provided to the metrics.

Reply:
thanks, https://github.com/thoth-station/investigator/pull/206/commits/f7fe6a4ea2fa85773b15cad3d26d883cbd7630c0, I did for all 3 workflows which are solver, revsolver and si.